src/mono/mono/mini/interp/jiterpreter.c

```diff
@@ -1645,8 +1646,9 @@ get_queue (int queue) {
 	GPtrArray *result = NULL;
 	if ((result = (GPtrArray *)mono_native_tls_get_value (key)) == NULL) {
 		g_assert (mono_native_tls_set_value (key, result = g_ptr_array_new ()));
-		mono_os_mutex_lock (&queue_mutex);
 		// WARNING: Ensure we do not call into the runtime or JS while holding this mutex!
```

Comment:
Just a tiny style comment. In the rest of the code this comment is after the `mono_os_mutex_lock` call.